src/amo/components/AddonMeta/index.js

```diff
@@ -58,19 +73,12 @@ export class AddonMetaBase extends React.Component<Props> {
               title: userTitle,
             },
             {
-              content: reviewCount,
-              title: reviewTitle,
+              content: makeLink(reviewCount),
+              title: makeLink(reviewTitle),
             },
             {
-              content: (
-                <Rating
```

Comment:
When you removed this `Rating` element that caused the stars to disappear from  the screen altogether and be replaced with a number, which is not what we want. The screen should look exactly as it did before, except that the items should be linked to the reviews page.

Reply:
The CSS issue is noted. Will take care of it. I was trying to figure out how to properly do the testing before I mentioned anything about it, since we are on that topic would you mind guiding me in the right direction on how I would go about implementing that. Once again I appreciate your feedback as well as time and effort.